src/helpers/figures/charts/abstract_chart.ts

```diff
@@ -23,11 +23,13 @@ export abstract class AbstractChart {
   readonly title: string;
   abstract readonly type: ChartType;
   protected readonly getters: CoreGetters;
+  readonly extraData?: any;
```

Comment:
```suggestion
  readonly extraData?: {};
```
I think we could force it to be an object at least. We should also document this in the doc but I won't push for it right now, being the worst example to follow for this...